sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/implementation/notifications/NotificationUtils.java

```diff
@@ -218,12 +241,15 @@ public static OffsetDateTime parseEpochTime(Long epochMilli) {
             ZoneId.of("UTC"));
     }
 
-    public static JSONObject getJSONObject(JSONObject payload, String property) throws JSONException {
-        return new JSONObject(payload.getString(property));
-    }
+    public static CloudType getUserCloudTypeFromSkypeToken(String skypeToken) {
+        String skypeId = decodeSkypeIdFromJwtToken(skypeToken);
+        String prefix = skypeId.substring(0, skypeId.indexOf(":"));
 
-    public static JSONArray getJSONArray(JSONObject payload, String property) throws JSONException {
-        return new JSONArray(payload.getString(property));
```

Comment:
Is this no longer required?

Reply:
No one calls these 2 functions. This was used to parse incoming notification payload. This functionality is replaced by Jackson & annotations.